src/leaflike/bookmarks/views.clj

```diff
@@ -90,6 +91,10 @@
         [:td [:a {:href (str "/bookmarks/edit/" (:id bookmark))
                   :bookmark_id (str (:id bookmark))}
               [:button.btn.btn-sm.btn-outline-secondary "Edit"]]]
+        [:td (f/form-to {:role "form"}
+                        [:post (str "/bookmarks/delete/" (:id bookmark))]
```

Comment:
Perhaps `bookmarks/:id/delete` would have been more intuitive? But that'd apply to other bookmark-related operations too e.g. `edit`.